src/NServiceBus.Core.Tests/AssemblyScanner/AssemblyScannerTests.cs

```diff
@@ -119,7 +119,7 @@ public void Assemblies_which_reference_older_nsb_version_are_included()
 
         [Test]
         [RunInApplicationDomain]
-        public void Assemblies_with_transitive_references_are_included()
+        public void Assemblies_with_transitive_core_references_are_included()
         {
             var busAssembly = new DynamicAssembly("Fake.NServiceBus.Core");
```

Comment:
It seems we almost always do this pattern with a few exceptions:

``` 
var busAssembly = new DynamicAssembly("Fake.NServiceBus.Core");
var scanner = new AssemblyScanner(DynamicAssembly.TestAssemblyDirectory)
{
   CoreAssemblyName = busAssembly.DynamicName
};
```

Would it be worthwhile to do that part in some kind of a setup step and then only show in the test the actual differen settings? And while we are at it instead of naming it `busAssembly` should we call it `coreAssembly`? Given that this is a minor comment and doesn't necessarily have to be addressed I'm still approving the PR.

Reply:
@danielmarbach done, can you review and re-approve if you're happy with it?